lib/self-evict.js

```diff
@@ -178,6 +178,24 @@ SelfEvict.prototype._preEvict = function _preEvict() {
     });
 };
 
+SelfEvict.prototype._getMembersToPing = function(ringpop) {
```

Comment:
Nit: I think `ringpop` can be obtained from the object via `this.ringpop`. What is the benefit of passing it in?